SELF_EVALUATION.md

```diff
@@ -47,12 +47,13 @@ Briefly reflect in writing on your experience solving this exercise. Just a
 sentence or two in response to each question is plenty.
 
 **What did you miss? What did you wish you did better?**
-
+// While testing, I noticed that a lot of my mistakes came from misspelling my variables. Good thing that they're always underlined so I remember to change them.
 
 
 **What challenges did you face, and how did you overcome them?**
-
+// I feel like I've gotten better at understanding the placement of the curly brackets and parentheses, but it is still trickt for me, it feels a little counterintuitive.
```

Comment:
Practice looking at correctly formatted code, and it will become more and more intuitive with experience.

By the way, not that important, but: you don't have to put the `//` here because this file is not Java code. (The file extension is `.md`, which means “[Markdown](https://www.markdownguide.org).”)